src/test/store/receive-profile.test.js

```diff
@@ -1,6 +1,7 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
```

Comment:
I originally had these tests in here, it didn't seem worth removing `// @flow` after types working.